schemas/payload.json

```diff
@@ -38,6 +38,10 @@
   ],
   "additionalProperties": false,
   "properties": {
+    "log": {
+      "title": "Custom log location",
+      "type": "string"
```

Comment:
I don't see any changes to actually use this value -- was this diff included by mistake?

If not, a `description` property would be great: What does this mean?  Is it a path on the host or in the docker container?  Or is it the artifact name?

Reply:
This is part of unit tests fixes broken in #409

Reply:
Ah, ok -- could you add the description then?  I didn't know such a thing was available (I still don't!)